oeps/oep-0045-arch-ops-and-config.rst

```diff
@@ -0,0 +1,175 @@
+
+Since defaults are provided by the application, many smaller deployments should not need to do much more than provide the required settings to operate. For development environments the config will likely change the defaults to more development appropriate values, e.g. debug settings, log levels, email settings etc.
+ 
+**Config file generation & management**
+
+Due to the varied needs and processes of different operators, how the config files are created, managed, or otherwise end up on the server is up to the operator and will depend greatly on their deployment strategy.
+ 
+**Documentation of settings**
+
+The settings found in both the ``required.py`` and ``defaults.py`` files will be documented to describe what they are and how they should be used. The documentation will consist of `Sphinx autodoc`_ compatible comments before each setting. For reference that is a ``“comment with special formatting (using a #: to start the comment instead of just #)”``. This keeps documentation close to the code as it is being written, while allowing it to be surfaced in generated docs.
+ 
+
+.. _Sphinx autodoc: https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directive-autoattribute
+
+Operations Manuals
+******************
+
+A clear manual of operations will exist in the form of RST files in an ``operations`` directory within the ``documentation`` directory for that application. See `this commit`_ for an example provided by the Open edX Build-Test-Release working group. The operations docs will cover common operations such as how to run the application for web traffic or as an async worker and how to manage the applications underlying database schema. It will also include a list of potential maintenance tasks operators may want to leverage such as clearing sessions or applying security patches. Finally it will include the list of ad-hoc management commands operators can use to help handle edge case or one-time operations.
+ 
+In the same vein as not dictating how operators create and manage their application config files, so too will operators be expected to manage how they execute the operations documented in the manual.
+
+.. _this commit: https://github.com/openedx-btr-wg/edx-platform/commit/        18effd83f983f497ca0a1535108fa41dc50d06a2#diff-ca02329742db0a77612a18ba1260d178R1-R39
+
+
+Alternatives Considered
+=======================
+
+Refactor configuration
+**********************
+
+Refactoring the configuration repo to reduce the amount of shared code and making it easier to read and understand what is being run when you execute a particular playbook.
+
+This approach was rejected due to the sheer volume of work required to make this change in an environment that is inherently difficult to test. Also because while it would improve ease of use, it would only provide parity in terms of functionality. Moving to containers will also improve ease of use as well as unlocking many potential future enhancements.
```

Comment:
Nit: 
"Also because while" -> "Also, while"
"Moving to containers will also improve ease of use as well as unlocking" -> "Moving to containers will also improve ease of use and unlock"